docs/eventing/broker-trigger.md

```diff
@@ -13,29 +13,63 @@ A Broker represents an 'event mesh'. Events are sent to the Broker's ingress and
 are then sent to any subscribers that are interested in that event. Once inside
 a Broker, all metadata other than the CloudEvent is stripped away (e.g. unless
 set as a CloudEvent attribute, there is no concept of how this event entered the
-Broker).
+Broker). There can be different classes of `Broker`s providing different kinds
+of semantics around durability of events, performance, etc. The `Broker` that is
+part of the Knative Eventing repo is used for these examples, it uses Knative
+[Channels](./channels/) for delivering events. Simple example showing a `Broker`
```

Comment:
"Simple example showing..." isn't a complete sentence. You could swap the following sentence "You can read more..." before this one, then add a colon at the end, like

> it uses Knative [Channels](./channels/) for delivering events. You can read more details about
[Channel Based Broker](./channel-based-broker.md). Simple example showing a `Broker` where the configuration is specified in a `ConfigMap` config-br-default-channel, which uses `InMemoryChannel`: